doc/version-history.rst

```diff
@@ -10,6 +10,11 @@ Version History
 .. No new work should be required in order to complete this section.
 .. Below is an example of a version history format.
 
+v0.4.0
+======
+* Swap ts-salobj base case.
+* Fix import path for pymodbus.
+
```

Comment:
I think you did more for this project, didn't you? Like add the sshtunnel infrastructure and more. Please add that here as well.